DDDetectors/src/Mask_o1_v01_geo.cpp

```diff
@@ -125,17 +146,29 @@ static dd4hep::Ref_t create_element(dd4hep::Detector& description,
 	transformer = Transform3D(RotationY(rotateAngle), RotateY( Position(0, 0, zPosition), rotateAngle) );
 	transmirror = Transform3D(RotationY(mirrorAngle), RotateY( Position(0, 0, zPosition), mirrorAngle) );
       }
+      
       // solid for the tube (including vacuum and wall): a solid cone
       ConeSegment tubeSolid( zHalf, rInnerStart, rOuterStart, rInnerEnd, rOuterEnd , phi1, phi2);
-
+      
       // tube consists of vacuum
-      Volume tubeLog( volName, tubeSolid, material ) ;
-      tubeLog.setVisAttributes(description, xmlMask.visStr() );
+      Volume tubeLog0( volName, tubeSolid, material ) ;
+      Volume tubeLog1( volName, tubeSolid, material ) ;
+      if (isSensitive) {
+	tubeLog0.setSensitiveDetector(sens);
```

Comment:
Indentation is still of for these, probably a tab vs spaces issue?

Reply:
you mean the lines inside the brackets? they are tabbed

Reply:
yes, and they should be using spaces.

Reply:
ok I did not know, fixed! There are also other IFs in the original version which use tabs instead of spaces (e.g. L145, L198, ...) should I change also those?

Reply:
No, only those lines you touched or created.

Reply:
Ok perfect, thanks! May I ask why though? Is it just to avoid different editors to interpret the tab differently?

Reply:
yeah, different editor settings interpreting tabs differently, and also, as you saw, the github code display.

Reply:
good, thank you!